en/debug/probe_stlink.md

```diff
@@ -0,0 +1,197 @@
+# STLink Debug Probe
+
+The [STLinkv3-MINIE][stlinkv3] is a cheap, fast and highly capable debug probe that can serve as a stand-alone debug and console communicator for a PX4 developer:
+
+- Just one single USB-C connection for Reset, SWD, SWO, and serial in a very small package!
+- Up to 24MHz SWD and SWO connection. Up to 16 MBaud serial. 1.65V to 3.6V target voltage. USB2 high-speed 480 Mbps connection.
+- Driven by STLink or OpenOCD software with wide device support.
+- Much cheaper (<15€) than a Pixhawk Debug Adapter (~20€) with a JLink EDU mini (~55€) or JLink BASE (~400€) while having better hardware specs.
+
+:::note
+There are no "out of the box" connectors for using the STLink Debug Probe with Pixhawk flight controllers.
+The [Pixhawk Debug Port Adapter](#pixhawk-debug-port-adapter) section below explains how you can create your own (some soldering required).
+:::
+
+## Debugging Configuration
```

Comment:
Soldering is a bit off-putting for many, and affects cost - its a lot more to use this if you don't have soldering kit/experience. I've added this as a note above, and separated the usage instructions under this heading "Debugging Configuration" - better name welcome!